### ISSUE TITLE
[Retail Bug Report] Can't open bag

### ISSUE THREAD

**Issue by MourierH** [tukui-org/ElvUI#580]
### What is the issue you are having?

When I press my keybind to open my bag that doesn't work
It worked again when I disabled the "Pawn" addon
I guess there is a conflict between Pawn and Elvui

(Error below)

### What is the expected behavior?

To open my bag 

### What actually happened?

Nothing happens

### Suggested Solution/Workaround

Conflict with pawn addon

### Errors

```shell
I got this error :

Message: Interface/AddOns/Pawn/Pawn.lua:422: attempt to call global 'GetContainerItemInfo' (a nil value)
Time: Wed Nov 16 19:13:24 2022
Count: 6
Stack: Interface/AddOns/Pawn/Pawn.lua:422: attempt to call global 'GetContainerItemInfo' (a nil value)
[string "=[C]"]: in function `GetContainerItemInfo'
[string "@Interface/AddOns/Pawn/Pawn.lua"]:422: in function `PawnIsContainerItemAnUpgrade'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:429: in function `UpdateItemUpgradeIcon'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:655: in function `UpdateSlot'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:683: in function `UpdateBagSlots'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:399: in function `UpdateAllSlots'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:2150: in function `OpenBags'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:2072: in function <Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua:2058>
[string "=(tail call)"]: ?
[string "=[C]"]: in function `ToggleBag'
[string "@Interface/FrameXML/ContainerFrame.lua"]:107: in function `ToggleBackpack_Individual'
[string "@Interface/FrameXML/ContainerFrame.lua"]:119: in function <Interface/FrameXML/ContainerFrame.lua:111>
[string "=[C]"]: in function `ToggleBackpack'
[string "@Interface/FrameXML/ContainerFrame.lua"]:290: in function `OpenBackpack'
[string "@Interface/FrameXML/ContainerFrame.lua"]:1826: in function <Interface/FrameXML/ContainerFrame.lua:1825>
[string "@Interface/FrameXML/ContainerFrame.lua"]:1888: in function <Interface/FrameXML/ContainerFrame.lua:1859>
[string "=[C]"]: in function `ToggleAllBags'
[string "OPENALLBAGS"]:1: in function <[string "OPENALLBAGS"]:1>

Locals:
```

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://www.tukui.org/download.php?ui=elvui&changelog).
- [X] I have verfied ElvUI is up to date by checking /estatus.
- [X] I have verfied my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by 1josh13** [comment]
I have this issue when Bagnon is enabled. I've disabled both ElvUI skin for bags and bag control in ElvUI settings, only fix is to disable Bagnon. However, with Pawn enabled (and bagnon disabled) bags work fine for me, at least with ELVUI bag settings disabled.

**Comment by Toasticon** [comment]
Pawn + ElvUI Bags works now with the upstream fix. Just download the update from code tab. I sadly doubt Bagnon will be update till release. And many thanks to the devs for adding blizz item sorting <3.

**Comment by kashyn89** [comment]
Same for me with pawn and elvui, is there a chance that this gets fixed if its in topic of elvui?

**Comment by Meojifo** [comment]
The bug seems appear when you have an item in the bag which can provide an upgrade. On an alt without any stuff in bags, the bags are correctly show

**Comment by KristopherRussell** [comment]
I have had the same issues. But, when I enable 'bag' under Elvui after opening the default bag, the issue remains the same. Bags were working pre 11.15.22 patch. All I do is disable bags and check this bug report in hopes the issue is resolved.

**Comment by ecojud** [comment]
I am using ElvUI and Pawn as mentioned by others.  I'm not using any other bag managing addon that I am aware of.  When I used my keybind (B) to open my bags, no bags were appearing.  Disabling bags in Elvui allowed the blizzard ui bags to display on keybind as expected.

After reading some comments here, I re-enabled ElvUI bags and troubleshot by disabling all General Options.  This resolved the problem temporarily and my bag display began to toggle as expected with my keybind.  I re-enabled the General Options and the bag display continued to work.  I tested on another character and found that disabling just "Upgrade Icon" in the General Options was enough to resolve the issue temporarily.  I can re-enable Upgrade Icon and still have functioning bag displays.  

However, the problem returns on /reload or logging out.  I have not tested other load screens like instance loading for dungeons or raids.
